rubygem-ffi/rubygem-ffi.spec

```diff
@@ -1,25 +1,42 @@
+# This package contains macros that provide functionality relating to
+# Software Collections. These macros are not used in default
+# Fedora builds, and should not be blindly copied or enabled.
+# Specifically, the "scl" macro must not be defined in official Fedora
+# builds. For more information, see:
+# http://docs.fedoraproject.org/en-US/Fedora_Contributor_Documentation
+# /1/html/Software_Collections_Guide/index.html
+
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global gem_name ffi
 %{!?enable_test: %global enable_test 0}
 
-Name:           rubygem-%{gem_name}
+Name:           %{?scl_prefix}rubygem-%{gem_name}
 Version:        1.4.0
-Release:        2%{?dist}
+Release:        3%{?dist}
```

Comment:
either keep it at '2' and we'll run tito tag which'll bump it, or bump to '3' and add a %changelog